anw/TD-ANWv2024.1.md

```diff
@@ -0,0 +1,746 @@
+    * [Medewerker haalt gegevens op bij het bronsysteem](#medewerker-haalt-gegevens-op-bij-het-bronsysteem)
+      * [Voorbeeld task](#voorbeeld-task)
+      * [Voorbeeld update task](#voorbeeld-update-task)
+      * [Voorbeeld verifiable credential Regisseur (get/put op task)](#voorbeeld-verifiable-credential-regisseur-getput-op-task)
+        * [Bronhouder](#bronhouder)
+        * [Zorgverlener](#zorgverlener)
+      * [Voorbeeld verifiable credential bronhouder (informatie opvragen)](#voorbeeld-verifiable-credential-bronhouder-informatie-opvragen)
+      * [Endpoints voor data overdracht](#endpoints-voor-data-overdracht)
+      * [Sequentie diagram](#sequentie-diagram-1)
+        * [Toelichting per stap](#toelichting-per-stap)
+  * [Logging](#logging)
+  * [Nuts register](#nuts-register)
+    * [ANW-Regisseur](#anw-regisseur)
+    * [ANW-Bronhouder](#anw-bronhouder)
+    * [ANW-Zorgverlener](#anw-zorgverlener)
+    * [TODO](#todo)
+<!-- TOC -->
+
+## Uitgeschreven proces flow
+
+### Uitgangspunten
+
+* De regisseur is bekend van de medewerker planning. Dit kan door middel van Excel lijsten die wekelijks gedeeld worden.
+  Dit is zoals het nu al gebeurd. De implementatie die in dit document beschreven wordt, houdt geen rekening met het
+  ontsluiten van de planning.
+* Er is een samenwerkingsverband tussen de regisseur en de zorgaanbieders.
+* Wij maken gebruik van directe aanroepen en niet van compositions. Hiervoor is gekozen omdat er “live” data
+  nodig is voor de ANW-usecase.
+
+### Woordenlijst
+
+* **Regisseur**: De partij die de koppeling tussen de cliënten en ANW-zorgmedewerker maakt.
+* **Bronsysteem**: Het systeem/zorgaanbieder waar de cliënt in bekend is.
+* **Ontvanger**: Het systeem/zorgaanbieder waar de ANW zorgmedewerker werkt.
+* **ANW-Bronhouder**: is het bronsysteem
+* **ANW-Zorgverlener**: is de ontvanger
+* **ANW-regisseur**: is de regisseur
+* **VC**: Verifiable credential
+* **VP**: Verifiable presentation
+
+### NUTS-adresboek
+
+De regisseur heeft toegang tot het NUTS-adresboek, hierin kunnen de leveranciers voor elke organisatie die onderdeel uit
+zal maken het ANW-netwerk aanduiden dat de betreffende organisatie gebruik maakt van de usecase “ANW-Zorgverlener”
+en “ANW-Bronhouder”. De regisseur is zelf ook te vinden in het adresboek voor de bronhouder en zorgverlener om toegang
+te geven tot deze regisseur met als service “ANW-Regisseur”.
+
+### Regisseur vraagt de cliënt / medewerker gegevens op in alle ECD’s
+
+#### Omschrijving
+
+De regisseur heeft een functionaliteit tot zijn beschikking waarmee hij eenvoudig cliënten en medewerkers met elkaar kan
+koppelen. Dit kan zowel zijn eigen ECD zijn als een applicatie van derden, zie onderstaande afbeelding.
+
+![][image1]  
+Het uitgangspunt is dat de regisseur een eigen NUTS-node heeft draaien. Deze node wordt “getrust” door de NUTS-nodes van
+de verschillende leveranciers. De regisseur kan met behulp van zijn eigen node een JWT-token opvragen om hiermee de
```

Comment:
> Deze node wordt “getrust” door de NUTS-nodes van
de verschillende leveranciers.

trust in another party is only needed when that party is an issuer of credentials other than authCreds.